src/ol_openedx_rapid_response/README.rst

```diff
@@ -0,0 +1,67 @@
+ol_openedx_canvas_integration
```

Comment:
Point #1,2 Addressed

Re: Point#3 - Recently we have been creating our basic architecture from edX's [cookie cutters](https://github.com/edx/edx-cookiecutters). edX has used now the `.rst` format for the repos. Since these plugins are purely going to be used for the open community of edX so I think it would be good if we keep on following the common Ope edX's approach.